client/components/Index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import {Redirect} from 'react-router-dom'
+
+const Index = props => {
+    return <Redirect to="/pets" />
+}
+
+export default Index

```

Comment:
After updating `rootRouter` with the correct redirect response we can also remove this component